src/identity/interaction/CompletingInteractionHandler.ts

```diff
@@ -0,0 +1,43 @@
+import { BadRequestHttpError } from '../../util/errors/BadRequestHttpError';
+import { FoundHttpError } from '../../util/errors/FoundHttpError';
+import type { InteractionHandlerInput } from './InteractionHandler';
+import { InteractionHandler } from './InteractionHandler';
+import type { InteractionCompleterInput, InteractionCompleter } from './util/InteractionCompleter';
+
+/**
+ * Abstract class for {@link InteractionHandler}s that need to call an {@link InteractionCompleter}.
+ * Calls the InteractionCompleter with the results returned by the helper function
+ * and throw a corresponding {@link FoundHttpError}.
```

Comment:
Can we have a bit more background about what a handler and a completer are?
Right now, the comment completely depends on the reader understanding the concepts of interaction handlers and completers; even though those concepts are linked, can we make the description a little more self-sufficient?

For example: why would an interaction handler call a completer? What kinds of handlers would call completers? etc.